app/views/pages/hiring_content.html.haml

```diff
@@ -0,0 +1,145 @@
+#page-hiring-job.bg-backgroundColorA.pb-16
+  %header.c-shapes
+    .md-container
+      = link_to hiring_page_path, class: "c-prominent-link" do
+        = graphical_icon "arrow-left"
+        Back to All Job Listings
+
+      .role
+        = graphical_icon "hiring-fundraiser", category: 'graphics', css_class: 'graphic'
+        .information
+          %h1.text-h0.mb-12 Content Factotum
+          .details
+            .remote-tag
+              = graphical_icon "stars"
+              100% remote
+            .salary Salary: £50k (~$68k, ~€60k)
+
+  %article
+    .sm-container
+      %p.text-p-2xlarge.mb-40
+        %strong Become Exercism's Content Factotum!
+        We need someone to lead and support our team of content-creating volunteers as we create one of the best educational resources on the planet.
+        Are you looking for a job that's a mixture of strategy, authoring, editing and leading, this might be the perfect role for you?
```

Comment:
Ending "this might be the role for you" with a question mark seems a bit odd to me.